ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -194,16 +191,15 @@ Item {
             }
 
             onClicked: {
-                newMessages = 0
                 scrollDownButton.visible = false
-                chatLogView.scrollToBottom(true)
+                chatLogView.positionViewAtBeginning()
             }
 
             StyledText {
                 id: nbMessages
-                visible: newMessages > 0
-                width: visible ? implicitWidth : 0
-                text: newMessages
+                visible: false
```

Comment:
I guess this will be added back in another PR?

Reply:
Yes. I removed it completely for now, it didn't work before anyway. This count will have a different meaning with current designs. Will implement it with followups.